lib/home/users/manage_users.dart

```diff
@@ -1,68 +1,46 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
+import 'package:thc/firebase/firebase.dart';
 
-class ManageUsers extends StatefulWidget {
+class ManageUsers extends StatelessWidget {
   const ManageUsers({super.key});
 
-  @override
-  State<ManageUsers> createState() => _ManageUsersState();
-}
-
-class _ManageUsersState extends State<ManageUsers> {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: const Text('User Management'),
+    final dataTable = StreamBuilder<QuerySnapshot>(
+      stream: db.collection('users').snapshots(),
+      builder: (context, snapshot) => DataTable(
+        sortColumnIndex: 0,
+        columns: const [
+          DataColumn(label: Text('Id')),
+          DataColumn(label: Text('Name')),
+          DataColumn(label: Text('Type')),
+          DataColumn(label: Text('Actions'))
+        ],
+        rows: [
+          if (snapshot.data?.docs.reversed case final users?)
+            for (final user in users)
+              DataRow(cells: [
+                DataCell(Text(user['id'])),
+                DataCell(Text(user['name'])),
+                DataCell(Text(user['type'])),
+                const DataCell(Text(''), placeholder: true),
+              ]),
+        ],
       ),
-      body: Padding(
-        padding: const EdgeInsets.fromLTRB(10, 0, 10, 0),
-        child: StreamBuilder<QuerySnapshot>(
-          stream: FirebaseFirestore.instance.collection('users').snapshots(),
-          builder: (context, snapshot) {
-            final List<DataRow> userWidgets = [];
-            if (snapshot.hasData) {
-              final users = snapshot.data?.docs.reversed.toList();
-              for (var user in users!) {
-                final userWidget = DataRow(selected: true, cells: [
-                  DataCell(Text(user['id'])),
-                  DataCell(Text(user['name'])),
-                  DataCell(Text(user['type'])),
-                  const DataCell(Text(''), placeholder: true),
-                ]);
-                userWidgets.add(userWidget);
-              }
-            }
-            return Expanded(
```

Comment:
When I opened this page on my device, I saw an exception thrown in the debug console:

```
══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
The following assertion was thrown while applying parent data.:
Incorrect use of ParentDataWidget.
The ParentDataWidget Expanded(flex: 1) wants to apply ParentData of type FlexParentData to a
RenderObject, which has been set up to accept ParentData of incompatible type BoxParentData.
```

`Expanded()` is made to fill space in a `Row` or `Column`; if you aren't in a row/column, you can used `SizedBox.expand()`.